packages/flutter/lib/src/rendering/object.dart

```diff
@@ -20,6 +20,10 @@ export 'package:flutter/foundation.dart' show FlutterError, InformationCollector
 export 'package:flutter/gestures.dart' show HitTestEntry, HitTestResult;
 export 'package:flutter/painting.dart';
 
+/// This is an experimental flag which should not be used in production and will
+/// be removed without warning.
+bool forceOpacityCompositing = true;
```

Comment:
Mark this as deprecated so people don't accidentally use it since the plan is to presumably remove this once migration is complete? That's what we usually do with these kind of flags that assist with migration.

Reply:
Also, can this just be a static on PaintingContext instead of a global to not expose this in the global namespace?